validator/accounts/v2/list.go

```diff
@@ -117,3 +131,82 @@ func listDirectKeymanagerAccounts(
 	fmt.Println("")
 	return nil
 }
+
+func listDerivedKeymanagerAccounts(
+	showDepositData bool,
+	wallet *Wallet,
+	keymanager *derived.Keymanager,
+) error {
+	au := aurora.NewAurora(true)
```

Comment:
Part of our cleanup should include making a global aurora instance, wdyt? Maybe making a `colors.go` file and doing `AccountsColor()`, `DirColor()` to make sure colors are consistently used.